packages/parrot-middleware/src/ParrotMiddleware.js

```diff
@@ -36,6 +37,9 @@ class ParrotMiddleware extends Parrot {
     if (typeof body === 'object') {
       res.json(body);
     } else if (typeof body === 'undefined') {
+      if (resKeys.includes('data') || resKeys.includes('errors')) {
+        res.json(response);
+      }
```

Comment:
These changes should return a partial success response (outside of {body: {}} object) for graphql calls